app/templates/chat-demo-postgres.html

```diff
@@ -13,12 +13,8 @@
     </p>
     </div>
     <div class="col-md-1">
-        <input type="checkbox" id="esv" name="esv" value="ESV-Bible" onchange="changeLabel(this)" checked>
-        <label for="esv">ESV Bible</label>
-    </div>
-    <div class="col-md-1">
-        <input type="checkbox" id="tw" name="tw" value="translation-words" onchange="changeLabel(this)">
-        <label for="tw">Translation Words</label>
+        <input type="checkbox" id="tyndale-open" name="tyndale-open" value="tyndale_open" onchange="changeLabel(this)" checked>
```

Comment:
Is there some way to dynamically populate these based on permissions?

Reply:
Yes, that would be good - maybe after the demo when we have multiple types of users.